### ISSUE TITLE
Portal - Apply the the Styling to the Default Arranger portal 

### ISSUE THREAD

**Issue by gokul472** [overture-stack/dms-ui#3]
## Objective
The objective of this ticket is to ensure that we apply the overall overture styling to the demo portal.

## Acceptance Criteria:

- [ ] The styling should be applied. The CSS for the overture branding to be applied to the DEMO portal. Like the Overture overall look and feel including the colours, font etc to be maintained.
- [ ] As the user select the values of the facets, the values under selection is shown in the query bar (as shown in design).
This will be dynamic as user changes the filter, the panel will be updated automatically.
- [ ] Query bar: If there is multiple selection of filter values, it will occupy the next line and continue.
There will be buttons for both "..." and LESS.

a) If the user has applied more than 2 values of the same facet, a more button will appear where user's can view all applied filters.
b) If the user has clicked on the "more" button to expand all values, that button will change to a "less" button, giving them the ability to collapse the values.

## Dev tasks

- [ ] To apply the styling to the Styling to the DEMO portal.
- [ ] create a new react app

![image.png](https://images.zenhubusercontent.com/5ec563ba889df3e7a41572b3/d6f185c7-68e4-48ab-9c95-2c3aaf0f5cf6)

![image.png](https://images.zenhubusercontent.com/5ec563ba889df3e7a41572b3/a84e067e-17dc-4ca2-b546-88a88d2a10c8)

## Zeplin Links

1. Basic logged out portal mockup: https://zpl.io/2jGGXBr
2. With columns dropdown open:https://zpl.io/bLMMR86
3. With table rows selected: https://zpl.io/VDyyMlW
4. Query bar when user searches: https://zpl.io/2ZWWG5l

## Wiki Link
https://wiki.oicr.on.ca/display/OV/UI+Specs+for+DMS+Portal#UISpecsforDMSPortal-LoggedOutView
refer to the point 6 in the table.

**Comment by b-f-chan** [comment]
Testing blocked waiting on #4 to get default metadata into Arranger, will carryover to Sprint 7

**Comment by b-f-chan** [comment]
Arranger setup complete, testing on QA unblocked. Staging still missing Arranger project, I will try and set that up myself today. But will do initial test on QA as well. Unblocking issue.

**Comment by b-f-chan** [comment]
Styling matches what is provided in Zeplin except for the following:

- Enabled checkbox state has a different colour and style than in the mockups, this is a limitation in Arranger and a workaround or hackaround may be required - Can be investigated in separate ticket

Smoke test on Staging to be done once release tag deployed

**Comment by rosibaj** [comment]
LGTM - @anncatton minimum height feedback added to future styling ticket #32.

**Comment by rosibaj** [comment]
@b-f-chan the checkbox enabled color/style is not a big deal.  we dont need to investigate that for future.
